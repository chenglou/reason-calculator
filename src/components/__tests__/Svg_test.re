open Jest;
open Expect;

let _ =

describe "Svg" (fun _ => {
  test "renders" (fun _ => {
    let children = [ <g />, <path /> ];
    let tree = Svg.createElement ::children () |> ReactShallowRenderer.renderWithRenderer;

    expect tree |> toMatchSnapshot;
  });
});
