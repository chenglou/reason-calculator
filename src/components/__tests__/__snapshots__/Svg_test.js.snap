// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Svg renders 1`] = `
<div
  className="css-1slixyj css-1df0zmg"
>
  <svg
    enableBackground="new 0 0 64 64"
    viewBox="0 0 64 64"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g />
    <path />
  </svg>
</div>
`;
